src/docfx/ErrorCodes.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Docs.Build
+{
+    internal static class ErrorCodes
+    {
+        public const string ConfigNotFound = "config-not-found";
```

Comment:
>ConfigNotFound [](start = 28, length = 14)

should the error code should have corresponding action or message formate? otherwise the error message is really hard to maintain and can be anything...